.github/workflows/irap-fedora-ci-runner.yaml

```diff
@@ -0,0 +1,69 @@
+name: Build and Push atlas-irap-env container
+
+on:
+  push:
+    branches: 
+      - master
+    paths:
+      - "irap-fedora/**"
+  pull_request:
+    paths:
+      - "irap-fedora/**"
+
+jobs:
+  test:
+    name: Test build
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: atlas-irap
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Read tag
+      id: gettag
+      run: echo "::set-output name=tag::$(head -n 1 irap-fedora/tag | awk -F':' '{print $2}')"
+
+    - name: Build image
+      id: build-image
+      uses: redhat-actions/buildah-build@v2
+      with:
+        image: ${{ env.IMAGE_NAME }}
+        tags: ${{ steps.gettag.outputs.tag }}
+        containerfiles: |
+          ./irap-fedora/Dockerfile
+          
+  # deploy:
+  #   name: Build and deploy image to quay.io/ebigxa
+  #   runs-on: ubuntu-20.04
+  #   env:
+  #     IMAGE_NAME: atlas-irap
+
+  #   steps:
+  #   - uses: actions/checkout@v2
+
+  #   - name: Read tag
+  #     id: gettag
+  #     run: echo "::set-output name=tag::$(head -n 1 irap-fedora/tag | awk -F':' '{print $2}')"
+
+  #   - name: Build image
+  #     id: build-image
+  #     uses: redhat-actions/buildah-build@v2
+  #     with:
+  #       image: ${{ env.IMAGE_NAME }}
+  #       tags: ${{ steps.gettag.outputs.tag }}
+  #       containerfiles: |
+  #         ./irap-fedora/Dockerfile
+    
+  #   - name: Push to Quay
```

Comment:
When you re-enable this, should it have a filter so it only happens with master commits? See e.g. https://github.com/ebi-gene-expression-group/atlas-containers/blob/40b787e88c7e4bb7a2407f05ae31ba3f849d3016/.github/workflows/conda-templated.yaml#L88

Reply:
Thanks for this, I'll add that filter!